lib/ex_format.ex

```diff
@@ -1,1371 +1,23 @@
 import Kernel, except: [to_string: 1]
+import ExFormat.Formatter, only: [to_string_with_comments: 1]
+import ExFormat.State
+import ExFormat.AST
+import ExFormat.Comments
+import ExFormat.Lines
```

Comment:
There is no performance loss whatsoever but it makes code very very hard to read because it becomes very hard to figure out where a function comes from.